Plugin/test1/build.gradle

```diff
@@ -1,29 +1,45 @@
 plugins {
-    id 'java'
-    id 'org.jetbrains.intellij' version '0.3.4'
+    id 'java-library'
+    id 'org.jetbrains.intellij' version '0.4.21'
 }
 
 group 'test'
-version '1.05-SNAPSHOT'
+version '1.7-SNAPSHOT'
 
 sourceCompatibility = 1.8
 
+// When this option is enabled and `ideDirectory` is set to Android Studio,
+// the build fails and generates the exception https://youtrack.jetbrains.com/issue/KT-32401
+// As a result of disabling building searchable options,
+// the configurables that your plugin provides won't be searchable in the Settings dialog.
+buildSearchableOptions.enabled = false
```

Comment:
I will check another way to solve this later. 
Since this feature is not essential at the moment, it should be ok to disable it